include/openthread/border_routing.h

```diff
@@ -403,6 +418,21 @@ otError otBorderRoutingGetNextPrefixTableEntry(otInstance
                                                otBorderRoutingPrefixTableIterator *aIterator,
                                                otBorderRoutingPrefixTableEntry    *aEntry);
 
+/**
+ * Iterates over the discovered router entries on infrastructure link.
```

Comment:
on an infrastructure link

Reply:
Changed it to "the infrastructure link"  (since it is set to a specific one when `BorderRouting` is first initialized).